src/main/java/org/jabref/logic/shared/PostgreSQLProcessor.java

```diff
@@ -89,7 +89,8 @@ public void startNotificationListener(DBMSSynchronizer dbmsSynchronizer) {
             // Otherwise the listener is going to be deleted by GC.
             PGConnection pgConnection = connection.unwrap(PGConnection.class);
             listener = new PostgresSQLNotificationListener(dbmsSynchronizer, pgConnection);
-            listener.start();
+            new Thread(listener, "PostgresSQLNotificationListener").start();
```

Comment:
Better reuse our threading infrastructure so that we are sure that the thread is shutdown on exit etc.